cpp/src/arrow/util/macros.h

```diff
@@ -18,6 +18,8 @@
 #ifndef ARROW_UTIL_MACROS_H
 #define ARROW_UTIL_MACROS_H
 
+#define STRINGIFY(x) #x
```

Comment:
Should be `ARROW_STRINGIFY`, IMHO. Unprefixed macros too easily cause conflicts and bugs with other libraries.

Reply:
At least 2 reviewers have asked for this - shall fix & update the commit 👍

Reply:
Fixed.